src/firecracker/src/api_server/request/hotplug/memory.rs

```diff
@@ -0,0 +1,69 @@
+// Copyright 2025 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use micro_http::Body;
+use vmm::logger::{IncMetric, METRICS};
+use vmm::rpc_interface::VmmAction;
+use vmm::vmm_config::memory_hotplug::MemoryHotplugConfig;
+
+use crate::api_server::parsed_request::{ParsedRequest, RequestError};
+
+pub(crate) fn parse_put_memory_hotplug(body: &Body) -> Result<ParsedRequest, RequestError> {
+    METRICS.put_api_requests.serial_count.inc();
```

Comment:
hehe, now you did the same copy/paste error as me (I am honored you're using my endpoint as a reference implementation though xP)